### PR TITLE
Check for size change instead of name for attachments already downloaded

### PR DESCRIPTION
**Pull request by Anaethelion**

### CODE REVIEW

File: geotrek/common/parsers.py [hunk 1]
```diff
@@ -466,6 +469,26 @@ def filter_attachments(self, src, val):
...
+                ftp.login(user=parsed_url.username, passwd=parsed_url.password)
+                ftp.cwd(directory)
+                size = ftp.size(parsed_url.path.split('/')[-1:][0])
+                return size == attachment.attachment_file.size
+
+            if parsed_url.scheme == 'http' or parsed_url.scheme == 'https':
+                http = urllib2.urlopen(url)
+                size = http.headers.getheader('content-length')
+                return size == attachment.attachment_file.size
+        except:
```
Comment (leplatrem) [inline geotrek/common/parsers.py]: Catch-all are bad

File: geotrek/common/parsers.py [hunk 2]
```diff
@@ -466,6 +469,26 @@ def filter_attachments(self, src, val):
...
+        try:
+            parsed_url = urlparse(url)
+            if parsed_url.scheme == 'ftp':
+                directory = dirname(parsed_url.path)
+
+                ftp = FTP(parsed_url.hostname)
+                ftp.login(user=parsed_url.username, passwd=parsed_url.password)
+                ftp.cwd(directory)
+                size = ftp.size(parsed_url.path.split('/')[-1:][0])
+                return size == attachment.attachment_file.size
```
Comment (leplatrem) [inline geotrek/common/parsers.py]: This will return true is size is the same. The fonction name is "has changed" though

**Comment by leplatrem** [comment]
Definitely would deserve tests.

File: geotrek/common/parsers.py (hunk 2 above)
Comment (gutard) [inline geotrek/common/parsers.py]: Yes, I already catched and fixed that.

Comment (leplatrem) [inline geotrek/common/parsers.py]: Grea! sorry then :( I didn't dedicate much time TBH

**Pull request merged**